lib/CPathfinder.cpp

```diff
@@ -19,6 +19,86 @@
 #include "CConfigHandler.h"
 #include "../lib/CPlayerState.h"
 
+bool canSeeObj(const CGObjectInstance * obj)
+{
+	/// Pathfinder should ignore placed events
+	return obj != nullptr && obj->ID != Obj::EVENT;
+}
+
+CNeighbourFinder::CNeighbourFinder(CPathfinder * pathfinder)
+	:pathfinder(pathfinder), neighbours(), neighbourTiles(), accessibleNeighbourTiles()
+{
```

Comment:
Circular dependency to pathfinder is also awkward. But we actually need CGameState and CpathNodeInfo here I think. Will think how to refactor it.